rds.tf

```diff
@@ -0,0 +1,42 @@
+data "aws_iam_policy_document" "rds_for_github" {
+
+  statement {
+    sid    = "AllowListDescribe"
+    effect = "Allow"
+    actions = [
+      "rds:Describe*",
+      "rds:ListTagsForResource",
+    ]
+    resources = ["*"]
+  }
+
+  statement {
+    sid    = "AllowGetOwn"
+    effect = "Allow"
+    actions = [
+      "rds:CreateDBSnapshot",
```

Comment:
- `rds:CreateDBSnapshot` requires `rds:AddTagsToResource` as an action
- `rds:ModifyDBInstance` requires `iam:PassRole` as an action for some changes

Reply:
- tags are copied automatically to snapshots, which so far has been enough for our users (checking chat history, we have had already manual backup/restores that worked fine)
 - https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance/blob/6b9b84fc1dd53033f76478cbc33f9726ad3a4933/main.tf#L186 doesn't have anything IAM and has worked so far, will revisit of course if requested